CenterForOpenScience/osf.io
# [Eng-None] Fix faker probabilistic failure for brand name

## Purpose

Currently travis tests for `test_context_data` will randomly fail because Faker makes brand name that's too long save. This fixes that. https://travis-ci.org/github/Johnetordoff/osf.io/jobs/728035724

## Changes

- gives brand name a longer name 

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify that brand names are still changable and don't break < 100 charecters
- Verify that nothing else has changed.

What are the areas of risk?

Low risk, tiny migration

Any concerns/considerations/questions that development raised?

## Documentation

🐞 fix, no new docs

## Side Effects

None that I know of.

## Ticket

None
